sidebars.js

```diff
@@ -92,61 +165,141 @@ const sidebars = {
               label: 'Database Setup Guides',
               collapsible: true,
               items: [
-                'scalar-kubernetes/SetupDatabaseForAWS',
-                'scalar-kubernetes/SetupDatabaseForAzure',
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/SetupDatabaseForAWS',
+                  label: 'AWS',
+                },
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/SetupDatabaseForAzure',
+                  label: 'Azure',
+                },
               ],
             },
             {
               type: 'category',
               label: 'Installation Guides',
               collapsible: true,
               items: [
-                'scalar-kubernetes/ProductionChecklistForScalarDLLedger',
-                'scalar-kubernetes/ProductionChecklistForScalarDLAuditor',
-                'scalar-kubernetes/AwsMarketplaceGuide',
-                'scalar-kubernetes/AzureMarketplaceGuide',
-                'scalar-kubernetes/CreateBastionServer',
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/ProductionChecklistForScalarDLLedger',
+                  label: 'Production checklist for ScalarDL Ledger',
+                },
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/ProductionChecklistForScalarDLAuditor',
+                  label: 'Production checklist for ScalarDL Auditor',
+                },
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/AwsMarketplaceGuide',
+                  label: 'Install from AWS Marketplace',
+                },
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/AzureMarketplaceGuide',
+                  label: 'Install from Azure Marketplace',
+                },
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/CreateBastionServer',
+                  label: 'Create a Bastion Server',
+                },
               ],
             },
             {
               type: 'category',
               label: 'Cluster Guides',
```

Comment:
I don't think `Cluster Guides` and `Container Image Guides` are needed in the sidebar because they are part of deployment and covered by the Deployment guides.

Reply:
Although I'm OK with not showing these docs in the sidebar in the future, as long as the content is available in the other docs, I do have one concern.

Many other docs link to these docs listed in this sub-category. If we remove these docs from the sidebar, when a visitor clicks a link to go to these pages, then the sidebar navigation will not be displayed on those pages. I think not showing the sidebar navigation would be confusing for users because they would need to go back to the previous page or go to the home page to see the sidebar navigation again.

With that in mind, how about we address this in a different PR?